xlive/H2MOD/Tags/MetaLoader/tag_loader.cpp

```diff
@@ -1349,7 +1349,7 @@ bool _cdecl LoadTagsandMapBases(int a)
 	///tag_injector testing
 	//Just for testing purpose,dont cluter here	
 	///Actual injection process after map load
-	if (tags::get_cache_header()->type != e_engine_type::SinglePlayerShared)
+	if (tags::get_cache_header()->is_single_player_shared())
```

Comment:
should be
if (!tags::get_cache_header()->is_single_player_shared())

Reply:
fixed